config/commands/log.yaml

```diff
@@ -3,19 +3,26 @@
 command:
   name: log
   description: log reflections
-  usage: log [-r | -r<num> <...reflection>]
+  usage: log [options] [response]
   options:
     -
-      name: reflection
+      name: retro
       abbr: r
-      help: Without a number argument, show list of retrospective questions for reflection. Provide a number to show the reflection question, and to log a reflection for the question.
+      help: Show list of retrospective questions for current retrospective. Use in conjunction with the `-q<id>` option to display and respond to retrospective questions.
+    -
+      name: question
+      abbr: q
+      help: Show a question with the id provided. Provide a response argument to log a response for the question.
```

Comment:
Clearly that thought came to me out of the blue, not because you wrote "the number provided is the index of the question in the retro survey"